### PR TITLE
add blueprint and organization stores

### PR DESCRIPTION
**Pull request by dooart**

### CODE REVIEW

File: packages/engine/src/index.ts [hunk 1]
```diff
@@ -113,6 +123,18 @@ export interface RagSearchOpts {
   bucketName?: string
 }
 
+export interface VectorStoreHook {
+  createEmbedding: (content: string) => Promise<Embedding>
+  delete: (key: string) => void
+  get: <T = unknown>(key: string, opts?: SoulStoreGetOpts) => Promise<(typeof opts extends { includeMetadata: true } ? VectorRecord : T) | undefined>
```
Comment (dooart) [inline packages/engine/src/index.ts:129]: we could make this one not async, but we'd have to preload all the content from organization and blueprint every time we're about to process a step. i feel like we shouldn't do that... thoughts?

Comment (dooart) [reply packages/engine/src/index.ts:129]: if we don't preload, then users will have to remember to use await when they're grabbing objects from blueprint/organization, unlike when they're getting from the soul store. i really don't like the idea of having this kind of exception in the api

one idea is to call it something other than `get` for the new stores

Comment (tobowers) [reply packages/engine/src/index.ts:129]: I really do thing we should await though. I'd be in favor of adding that to the soul store too as people seem to be dropping a lot of data into there. 

Maybe it's "fetch" here with await though?

Comment (dooart) [reply packages/engine/src/index.ts:129]: so deprecate get and introduce async fetch everywhere?

File: packages/engine/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opensouls/engine",
-  "version": "0.1.4",
+  "version": "0.1.5",
```
Comment (dooart) [inline packages/engine/package.json:3]: @tobowers should we bump the other packages too from 0.1.4 to 0.1.5?

Comment (tobowers) [reply packages/engine/package.json:3]: 👍 - I added a couple of repo commands to make that easier.

**Review by tobowers — APPROVED** [review]
w00t

**Comment by tobowers** [comment]
I think the double yjs import might have come from here.

**Review by tobowers — CHANGES_REQUESTED** [review]
switching this to request changes just because it's dangerous if we have the two yjs

**Pull request merged**